assets/src/domain/shared/entities/tickets/predicates/isSoldOut/index.ts

```diff
@@ -1,5 +1,12 @@
+import { is } from 'ramda';
+
 import { Ticket } from '@edtrServices/apollo/types';
+import { isValidOrTrashed } from '../../../../services/predicates';
+
+const isSoldOut = (ticket: Ticket): boolean => {
+	const { quantity, sold } = ticket;
 
-const isSoldOut = (ticket: Ticket): boolean => ticket.quantity <= ticket.sold;
+	return isValidOrTrashed(ticket) && is(Number, quantity) && isFinite(quantity) && quantity <= sold;
```

Comment:
`is(Number, quantity)` check is not necessary. In fact it can be a problem in certain cases when forms return numeric `string` instead of `number` and it's not handled properly.

Reply:
Removed that check.
It's interesting,  `quantity` is typed as numer in `Ticket` type, the form should not be able to process that string.